app/userland/settings/js/views/general.js

```diff
@@ -59,10 +58,64 @@ class GeneralSettingsView extends LitElement {
       ${this.renderRunBackgroundSettings()}
       ${this.renderNewTabSettings()}
       ${this.renderDefaultZoomSettings()}
+      ${this.renderSearchSettings()}
       ${this.renderProtocolSettings()}
       ${this.renderAnalyticsSettings()}
     `
   }
+  renderSearchSettings() {
+
+
+      return html`
+        <div class="section new-tab">
+          <h2 id='search-settings class='subtitle-heading>Default Search Settings</h2>
+            ${this.settings.default_search_engines.map((engine,i)=>{
+
+              return html`
+                        <div class="radio-item">
+                          <input type="radio"
+                                 id="engine${i}"
+                                 name="search-engine"
+                                 value="${i}"
+                                 ?checked="${this.settings.selected_search_engine === i}"
+                                 @change="${this.onSearchEngineChange}"/>
+                          <label for="engine${i}">
+                            ${engine.name} (${engine.url})
+                          </label>
+                        </div>`
```

Comment:
Tab depth is a little off on this. I'm okay with how you aligned the attributes but the markup chunks should be only 2 spaces deeper than the `return` statement